doc/man7/ssl.pod

```diff
@@ -89,6 +89,10 @@ includes both more private SSL headers and headers from the B<crypto> library.
 Whenever you need hard-core details on the internals of the SSL API, look
 inside this header file.
 
+ssl.h includes a macro OPENSSL_VERSION_AT_LEAST(maj,min,fix), which can be used in #if statements in order to determine which version of the library is
```

Comment:
Please word-wrap to shorter length.  I'm torn between where the longer description goes, but that's a side-note (not required change).

Reply:
Also, please change `#if` to `C<#if>` and `ssl.h` to `opensslv.h`